### name of the pose
string name

### goal pose values
geometry_msgs/Transform goal_pose
